gulpfile.js

```diff
@@ -260,6 +260,7 @@ const modules = gulp.parallel(
               '!**/__tests__/**',
               '!**/__flowtests__/**',
               '!**/__mocks__/**',
+              '!**/node_modules/**',
```

Comment:
I lost a bunch of time faffing around trying to find the best way to do continuous development on the package while integrated into one of our apps, which led me to at some point running `yarn install` in the `packages/relay-compiler` dir. After that building failed in confusing ways, so this addition is really just to guard other devs from the same silly mistake.